[2025-10-16T10:24:01.865+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-16T10:24:01.889+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T10:23:00+00:00 [queued]>
[2025-10-16T10:24:01.897+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T10:23:00+00:00 [queued]>
[2025-10-16T10:24:01.898+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-16T10:24:01.916+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dag2_bash_1> on 2025-10-16 10:23:00+00:00
[2025-10-16T10:24:01.927+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=360) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-16T10:24:01.929+0000] {standard_task_runner.py:63} INFO - Started process 362 to run task
[2025-10-16T10:24:01.928+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag2', 'dag2_bash_1', 'scheduled__2025-10-16T10:23:00+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpv4ypaeeq']
[2025-10-16T10:24:01.932+0000] {standard_task_runner.py:91} INFO - Job 92: Subtask dag2_bash_1
[2025-10-16T10:24:01.947+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-16T10:24:01.984+0000] {task_command.py:426} INFO - Running <TaskInstance: dag2.dag2_bash_1 scheduled__2025-10-16T10:23:00+00:00 [running]> on host bb861ccd12fb
[2025-10-16T10:24:02.079+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag2' AIRFLOW_CTX_TASK_ID='dag2_bash_1' AIRFLOW_CTX_EXECUTION_DATE='2025-10-16T10:23:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-16T10:23:00+00:00'
[2025-10-16T10:24:02.081+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-16T10:24:02.082+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-16T10:24:02.084+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "hello bash"']
[2025-10-16T10:24:02.099+0000] {subprocess.py:86} INFO - Output:
[2025-10-16T10:24:02.100+0000] {subprocess.py:93} INFO - hello bash
[2025-10-16T10:24:02.102+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-16T10:24:02.103+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-16T10:24:02.137+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag2, task_id=dag2_bash_1, run_id=scheduled__2025-10-16T10:23:00+00:00, execution_date=20251016T102300, start_date=20251016T102401, end_date=20251016T102402
[2025-10-16T10:24:02.189+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-16T10:24:02.210+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-16T10:24:02.213+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
